cartographer_ros/launch/assets_writer_backpack_2d_ci.launch

```diff
@@ -0,0 +1,27 @@
+<!--
+  Copyright 2016 The Cartographer Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<launch>
```

Comment:
Use the roslaunch `arg` feature to reduce duplication between launch files. See https://github.com/googlecartographer/cartographer_turtlebot/blob/master/cartographer_turtlebot/launch/turtlebot_depth_camera_2d.launch.